### DISCUSSION TITLE
Appdeploytoolkit and VBScript

### DISCUSSION THREAD

**Discussion by FRacine** [PSAppDeployToolkit/PSAppDeployToolkit#860] [Q&A] [answered]
As Microsoft announced VBScript is deprecated, how will Appdeploytoolkit moving?

**Comment by That-Annoying-Guy** [comment, marked as answer]
we are working to remove it.

**Reply by Fixfinder** [reply]
any update please? can we expect anytime soon for a new release without vbs?
for the meantime, we're still using 3.9.3, just so we don't have to rework our existing packages as well

**Comment by DanGough** [comment]
It’s scheduled for v4 which will be released later this year.
